v0/util/error.js

```diff
@@ -0,0 +1,64 @@
+function ErrorBuilder() {
```

Comment:
https://github.com/rudderlabs/rudder-transformer/blob/2aa28b7785858dcfe64c314f718eac8566050868/v0/util/response-builders.js

Why can't we write something like this ? Just wanted to know the reason you've implemented here

Reply:
Yeah since I was using the error part only I kept it here but yeah we can write like this